main/lint.xml

```diff
@@ -10,6 +10,9 @@
     <!-- false positives for return values of RxJava subscribe() calls -->
     <issue id="CheckResult" severity="ignore" />
 
+    <!-- avoid outdated lint warnings typicly from external libraries -->
+    <issue id="ObsoleteLintCustomCheck" severity="ignore" />
+
```

Comment:
Can we set the ignore flag for external libraries only?
Otherwise we would switch it off for our own code, too, which I'd like to avoid.

BTW: spelling is "typically"

Reply:
We do not use custom lint checks in c:geo, so there is no problem for the normal checks.
See https://ci.cgeo.org/view/Continous%20integration/job/cgeo%20continuous%20integration/3047/androidLintResult/type.-1692134117/

Will fix the typo :-)